stack.yaml

```diff
@@ -8,4 +8,7 @@ packages:
 - zifter-hindent
 - zifter-hlint
 - zifter-stack
+extra-deps:
+- validity-0.4.0.4
+- genvalidity-0.4.0.4
```

Comment:
validity is on stackage. Better to wait until the right version is available there.